rfcs/text/0004-dss-deletion-process.md

```diff
@@ -407,10 +421,10 @@ data that was permanently deleted over the previous 24 hours.
 
 ### Drawbacks and Limitations [optional]
 
-* This introduces the ability to permanently destroy data from the DSS. 
+* This introduces the ability to permanently destroy data from the DSS.
 * There is no limit on the amount of data that can be rapidly permanently deleted.
 * Permanently deletion does not occur until after the grace period has elapsed. This could
-  be problematic if files need to be removed sooner. 
+  be problematic if files need to be removed sooner.
 * If two bundles share files and only one bundle is deleted, this causes the remaining bundle to be incomplete.
```

Comment:
Is this how tombstoning currently works? almost all bundles share common files (e.g. project, donor) If a bundle is deleted, does that mean all other bundles wouldn't contain those shared files?

Reply:
Yes, everything bundle containing that file would be logically deleted.

Reply:
The logically deleted bundles containing physically deleted files can be resubmitted, minus the deleted files to fix the bundle.

Reply:
Just to check if my understanding is correct, if i have the following bundles and files:

Bundle 1 : 
- Project
- Donor
- Specimen
- Cell Suspension
- Sequencing File 1.1, Sequencing File 1.2, Sequencing File 1.3

Bundle 2: 
- Project
- Donor
- Specimen
- Cell Suspension 2
- Sequencing File 2.1 , Sequencing File 2.2, Sequencing 2.3
Bundle 3 : Project, Donor, Specimen, Cell Suspension 3, Sequencing File 3.1, Sequencing File 3.2, Sequencing 3.3

If I requested Bundle 1 to be tombstoned ( just "logically deleted"):
1. will all files get tombstoned?
2. will Bundle 2 and 3 be tombstoned as well? 

I don't know if this is a valid use case, but will it be possible for the user to delete Bundle 1 but not delete Project, Donor, Specimen?

Reply:
> 1. will all files get tombstoned?

No - if a bundle is logically deleted, files are unaffected.

> 2. will Bundle 2 and 3 be tombstoned as well?

No - Bundle 2 and Bundle 3 would be unaffected by a logical deletion of Bundle 1.

> I don't know if this is a valid use case, but will it be possible for the user to delete Bundle 1 but not delete Project, Donor, Specimen?

I think this is a valid use case. This can be accomplished by logically deleting the bundle. This logical deletion would also occur were a file only present in Bundle 1 physically deleted.

Reply:
I see, thanks!